py3status/modules/systemd_suspend_inhibitor.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""
+Turn on and off systemd suspend inhibitor.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module; for xfce4-notifyd
+        (default 30)
+    format: display format for this module
+        (default '[\?color=state&show SUSPEND [\?if=state OFF|ON]]')
+    thresholds: specify color thresholds to use
+        (default [(True, 'bad'), (False, 'good')])
+
+Format placeholders:
+    {state} systemd suspend inhibitor state, eg True, False
+
+Color thresholds:
+    xxx: print a color based on the value of `xxx` placeholder
+```
+
+
+Examples:
+# display SUSPEND ON/OFF
+systemd_suspend_inhibitor {
+    format = '[\?color=state&show SUSPEND [\?if=state OFF|ON]]'
+    thresholds = [(True, "bad"), (False, "good")]
+}
+```
+
+@author Cyrinux https://github.com/cyrinux
+@license BSD
+
+SAMPLE OUTPUT
+[{'full_text': 'SUSPEND ON', 'color': '#00FF00'}]
+[{'full_text': 'SUSPEND OFF', 'color': '#FF0000'}]
```

Comment:
This produces `SAMPLE DATA ERROR` here. I'm guessing you are missing a title for second screenshot.

Reply:
Still a `SAMPLE DATA ERROR`. Need a blank line.